action/protocol/staking/protocol.go

```diff
@@ -57,6 +60,12 @@ var (
 	TotalBucketKey     = append([]byte{_const}, []byte("totalBucket")...)
 )
 
+var (
+	_nameKey     = []byte("name")
+	_operatorKey = []byte("oper")
+	_ownerKey    = []byte("owne")
```

Comment:
zhi: using `names, operators, owners` won't take too many disk spaces

Reply:
right, disk space is not a concern here
I prefer for these meta keys to have the same length, similar to namespace in filedaov2

Reply:
im also confused here

Reply:
fixed